ts/kit/src/instructions/permission-program/updatePermission.ts

```
@@ -0,0 +1,53 @@
+import {
+  Address,
+  Instruction,
+  AccountMeta,
+  AccountRole,
+} from "@solana/kit";
+import { PERMISSION_PROGRAM_ID } from "../../constants";
+
+/**
+ * UpdatePermission instruction arguments
+ */
+export interface UpdatePermissionInstructionArgs {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)